cpp/include/cugraph_c/graph_functions.h

```diff
@@ -102,6 +102,26 @@ cugraph_error_code_t cugraph_two_hop_neighbors(
   cugraph_vertex_pairs_t** result,
   cugraph_error_t** error);
 
+/**
+ * @brief      Verify if a vertex exists in the graph
+ *
+ * @param [in]  handle         Handle for accessing resources
+ * @param [in]  graph          Pointer to graph
+ * @param [in]  vertex         Vertex to be queried
+ * @param [in]  do_expensive_check
+ *                             A flag to run expensive checks for input arguments (if set to true)
+ * @param [out] result         Opaque pointer to resulting queried vertex
+ * @param [out] error          Pointer to an error object storing details of any error.  Will
+ *                             be populated if error code is not CUGRAPH_SUCCESS
+ * @return error code
+ */
+cugraph_error_code_t cugraph_has_vertex(const cugraph_resource_handle_t* handle,
+                                        cugraph_graph_t* graph,
+                                        const int vertex,
```

Comment:
Can't use an int here.  A vertex can be int32_t or int64_t.

For SSSP, we pass in a `size_t` and cast it to either int32_t or int64_t inside the C API implementation.